docs/content.zh/docs/dev/configuration/maven.md

```diff
@@ -28,7 +28,7 @@ under the License.
 
 ## 要求
 
-- Maven 3.8.6 (推荐或者使用更高版本)
```

Comment:
In English version [README.md](https://github.com/apache/flink/blob/master/README.md#building-apache-flink-from-source), we only mention we require maven v3.8.6:

`Maven (we require version 3.8.6)`

Reply:
Thanks @reswqa ! I understand. 
So, in summary, the English version has no issue because it said `Maven (we require version 3.8.6)`, compared with Chinese version, it said `Maven 3.8.6 (recommended or higher)`.

Reply:
My bad. Didn't get it. Updated in https://github.com/apache/flink/pull/24863/commits/5d747e02e92655c18f53bcdce9e36457f9633f70.